utils/RASAero_fetch/src/script.py

```diff
@@ -0,0 +1,106 @@
+import os
+import pyautogui
+import time
+from pathlib import Path
+
+def main():
+
+    out_pth = os.getcwd()[:-3] + "output\\"
```

Comment:
why do you cut off the last 3 characters from the path, it makes the path wrong on my machine

Reply:
@rishivg Might know more about this